doc/dev/dev_setup.md

```diff
@@ -30,16 +30,14 @@ or execute the various commands available in the toolbox.
     ./env/scripts/activate.bat   # Windows CMD only
     ```
 
-4.  Setup your env (installing dependencies, etc.)
+4. Setup your development environment
 
+    Install the development requirements for a specific library (e.g. Azure Form Recognizer), [Tox](https://tox.readthedocs.io/en/latest/), and an editable install of your library:
```

Comment:
1. For clarity, can you include something like "Install the development requirements (located in the `dev_requirements.txt` file at the root of the library"
2. I think you should briefly mention tox monorepo as well

Love the complete walk through though!

Reply:
great catch on tox monorepo, completely forgot about that!